Configure

```diff
@@ -490,6 +490,7 @@ my @disablables = (
     "ssl",
     "ssl-trace",
     "static-engine",
+    "stats",
```

Comment:
Please document the new option in INSTALL.md. Since we are effectively changing this to be "default off", but in previous releases it was "default on" I think a mention in CHANGES.md is warranted.